results/demo/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func main() {
+	kubeconfig := flag.String("kubeconfig", "/root/.kube/config", "absolute path to the kubeconfig file")
```

Comment:
We shouldn't recommend the use of root. We should default to the user's home directory.

Reply:
e.g. `filepath.Join(os.Getenv("HOME"), ".kube", "config")`

Reply:
deleted it. but good advice, thanks!